### PR TITLE
scripts: Docopt in prci_test_control

### PR DESCRIPTION
**Pull request by celestian**
I rewrote prci_test_control to use docopt instead of argparse.

### CODE REVIEW

**Comment by celestian** [comment]
Thanks @davidkupka  for advice. I used your suggested patch and I fixed some pylint issues.

It could be helpful to me to interact more (in reviews) with experienced python engineers. I did not know how much my experience with programming in C changed my ability to write nice and readable python code.

**Comment by davidkupka** [comment]
@celestian I didn't want you to replace your work with my patch. I created and attached  the patch to demonstrate the difference in number of changes and readability (but readability is very subjective).

You're right, if you want to improve your python knowledge, interaction with experienced python developers helps a lot. But I think that the will to use python and contribute to project is more important than skill level.

I will pass review request to @tomaskrizek  because now I would be doing self-review :-)

**Comment by celestian** [comment]
@felipevolpone , please, could you do code review of this? It seems that @tomaskrizek is busy.

File: scripts/prci_test_control.py [hunk 1]
```diff
@@ -138,5 +140,7 @@ def cmd_rerun(self, args):
         tc(args)
     except ValueError as exc:
         print(exc)
-        parser.print_help()
         exit(2)
+
```
Comment (felipevolpone) [inline scripts/prci_test_control.py]: PEP 8 requires two empty lines between the end of `main` method and the `if` statement.

**Comment by felipevolpone** [comment]
Hi @celestian, it LGTM. 
I dropped a comment about a simple pep8 change, once you have it fixed, I'll merge it.
Thank you for your contribution.

**Comment by celestian** [comment]
Addressed.

**Pull request merged**